src/applications/pact-act/containers/questions/camp-lejeune/Lejeune-2-4.jsx

```diff
@@ -74,11 +74,11 @@ const mapDispatchToProps = {
 
 Lejeune24.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setLejeune24: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.